content/guides/hanami/v2.3/database/relations.md

```diff
@@ -281,7 +291,10 @@ Hanami supports querying the database using a **query builder** , a pattern that
 
 SQL query building is a _gigantic_ topic, so this will only scratch the suface of what is possible.
 
-ROM Relations are built on top of [Sequel Datasets](http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html). Inspect the generated SQL of a relation by calling `.dataset.sql` on it.
+<p class="notice">
+  ROM Relations are built on top of <a href="http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html">Sequel
+  Datasets</a>. Inspect the generated SQL of a relation by calling <code>.dataset.sql</code> on it.
+</p>
```

Comment:
For these callouts, we have special GFM syntax that we can use in this new site :)

The syntax is here: https://github.com/hanakai-rb/site/issues/74

And of course, @makenosound has already styled them up to look beautiful! Would you be OK to port this (and any other) callouts to that format?